docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -1486,11 +1486,18 @@ supported as a space-separated list. For example,
   depends_on:
   - name: test
     variant: "* !E"
+```
+
+Notably, selectors return items that satisfy all of the criteria. That is,
+they return the *set intersection* of each individual criterion. So the below yaml,
+while technically valid, wouldn't match anything given that these are static variant names, so the set 
+intersection will be nothing.
```

Comment:
Should we maybe add another example where the set intersection would be something?

Reply:
The example of "* !E" covers this case.